distribution/tools/upgrade-cli/build.gradle

```diff
@@ -21,7 +21,10 @@ dependencies {
   implementation "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
   testImplementation project(":test:framework")
   testImplementation 'com.google.jimfs:jimfs:1.2'
-  testRuntimeOnly 'com.google.guava:guava:32.0.1-jre'
+  testRuntimeOnly 'com.google.guava:guava:32.1.1-jre'
```

Comment:
Wow, I saw the error but test dependencies should never be brought up transitively, under any conditions.